docs/content/latest/architecture/transactions/distributed-txns.md

```diff
@@ -29,19 +29,18 @@ responsible for the keys the transaction is trying to modify. We call them "prov
 to "regular" ("permanent") records, because they are invisible to readers until the transaction
 commits.
 
-Provisional records are stored in a separate part of the RocksDB key space in the same RocksDB /
-DocDB instance as regular records, but with a separate prefix that puts all provisional records'
-RocksDB keys before those of regular records. Compared to other possible design options, such as
-storing provisional records inline with the regular records or putting them in a separate RocksDB
-instance altogether, the approach we have chosen has the following benefits:
+Provisional records are stored are stored in a separate RocksDB instance in the same tablet peer.
+Compared to other possible design options, such as storing provisional records inline with the
+regular records or putting them in the same RocksDB instance altogether with regular records, the
+approach we have chosen has the following benefits:
 
 - It is easy to scan all provisional records. As we will see, this is very helpful in cleaning up
     aborted / abandoned transactions.
 - During the read path, we need to handle provisional records very differently compared to regular
     records, and putting them in a separate section of the RocksDB key space allows to simplify the
     read path.
-- Storing provisional records in the same RocksDB instance allows to atomically delete provisional
```

Comment:
I thought we didn't call this RocksDB because DocDB is our enhanced version of RocksDB. Does calling it RocksDB add any value to understanding our product?

Reply:
According to https://docs.yugabyte.com/latest/architecture/docdb: "DocDB is YugabyteDB's distributed document store responsible for transactions, sharding, replication, and persistence". So it is a much larger piece than just an enhanced version of RocksDB.

Reply:
@mbautin @bmatican what do you think?

Reply:
It is fine to say RocksDB here because we are literally talking about separate RocksDB instances as part of the same DocDB instance in the tablet.